### ISSUE TITLE
System status appears as undecipherable characters in a Chinese environment

### ISSUE THREAD

**Issue by cesuogui** [webmin/webmin#2237]
![2024727-195218](https://github.com/user-attachments/assets/3a32302d-deba-4a4c-8c12-a017946b3ff7)

**Comment by jcameron** [comment]
Does this still happen if you switch to the Framed Theme at Webmin -> Change Language and Theme?

**Comment by cesuogui** [comment]
> Does this still happen if you switch to the Framed Theme at Webmin -> Change Language and Theme?

Yes, the language configuration is correct. When you first open it in the browser, it displays correctly. The garbled situation occurs after the system status is updated in real-time, such as changes in memory and CPU status

![1](https://github.com/user-attachments/assets/9f0a95f2-f6db-41bb-9b06-833ea08acb4e)

after real-time status update
![2](https://github.com/user-attachments/assets/1e2f4025-075f-4598-9c38-139cce3e6e59)

**Comment by iliajie** [comment]
Ni-hao!

Thanks for the heads up! It is a bug! It was fixed in the commit linked above!

The patch can be applied to your system by running:

```
webmin patch https://github.com/webmin/authentic-theme/commit/53bca06
```

**Comment by cesuogui** [comment]
thanks it has fix in new version [2.202]

**Comment by cesuogui** [comment]
thanks it has fix in new version in [2.202]

**Comment by iliajie** [comment]
I appreciate your feedback and I'm glad it's working for you now!
